utilities/cloudharness_utilities/deployment-configuration/helm/templates/configmap.yaml

```diff
@@ -0,0 +1,22 @@
+{{- /* Be careful with json typos here, mainly trailing ',' */}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.apps.accounts.name | quote }}
```

Comment:
Why an application-specific config map here?

Reply:
ow yes agree, this is a copy&paste error, will remove it